sdx_controller/controllers/l2vpn_controller.py

```diff
@@ -152,7 +152,8 @@ def place_connection(body):
     body["status"] = str(ConnectionStateMachine.State.REQUESTED)
 
     # used in lc_message_handler to count the oxp success response
-    body["oxp_success_count"] = 0
+    if "oxp_success_count" not in body:
+        body["oxp_success_count"] = 0
```

Comment:
@congwang09 since the place_connection method is only called from swagger route to POST /l2vpn, and moreover the schema defined on swagger does not allow this attribute `oxp_success_count`, this IF statement will be always true. That is why we just set this attribute to start with value zero.